documentation/release_3.1.htm

```diff
@@ -120,6 +125,8 @@ <h4>Python (and MATLAB) interface</h4>
   As opposed to <code>bin.view_num()</code>, write <code>bin.view_num</code>. You can now also do
   <code>bin.view_num=1</code>.
 </li>
+<li> <code>CListModeData</code> now need to implement a <code> ProjDataInfo</code> instead of <code>Scanner</code>. Therefore, acquisition information can be used directly in the LM reconstruction. 
```

Comment:
delete as also on L247. I can't see what you mean with "acquisition information can be used". It of course allows us to handle data where listmode data is not uncompressed (span=1 and mash=1), but with acquisition information I would understand `ExamInfo`